Documentation/dev/release.md

```diff
@@ -61,16 +60,14 @@ It generates all release binaries and images under directory ./release.
 
 ## Sign Binaries and Images
 
-Choose appropriate private key to sign the generated binaries and images.
+etcd project key must be used to sign the generated binaries and images.
 
 The following commands are used for public release sign:
 
 ```
 cd release
-# personal GPG is okay for now
-for i in etcd-*{.zip,.tar.gz}; do gpg --sign ${i}; done
-# use `CoreOS ACI Builder <release@coreos.com>` secret key
-gpg -u 88182190 -a --output etcd-${VERSION}-linux-amd64.aci.asc --detach-sig etcd-${VERSION}-linux-amd64.aci
+for i in etcd-*{.zip,.tar.gz}; do gpg2 --default-key $SUBKEYID --output ${i}.asc --detach-sign ${i}; done
```

Comment:
Just added more explanation. As Dalton explained, the whole process is documented in our private repository.

Thanks.